tests/e2e/utilities/setup-and-go.ts

```diff
@@ -34,6 +34,7 @@ export async function setupAndGo(
   },
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
 ): Promise<{ browser: any; messageArray: string[]; networkArray: string[] }> {
+  jest.retryTimes(3);
```

Comment:
E2E's are notoriously flaky. By leveraging the `jest/circus-runner` which is the default test runner in jest 27, we can use `retries` for tests that fail for reasons that are not because of code/env of ours. 

For example, the browser failing to spin up. 3 times should be enough but we can edit it and see how it looks.

Reply:
I'm okay with this, but I want to really stress that I want to avoid the view that E2E tests are inherently flaky. Flaky tests that can't be trusted _rapidly_ lose their value. If we see some instability in our tests, I want to evaluate the core problem and try to fix it, rather than just running the tests additional times.